ftplugin/vlang.vim

```diff
@@ -1,33 +1,25 @@
 if exists("b:did_ftplugin")
 	finish
 endif
-
-setlocal commentstring=//\ %s
-setlocal makeprg=v\ %
-
-if exists('b:undo_ftplugin')
-	let b:undo_ftplugin .= "|setlocal commentstring< makeprg<"
-else
-	let b:undo_ftplugin = "setlocal commentstring< makeprg<"
-endif
+let b:did_ftplugin = 1
```

Comment:
this should make the first if/finish work. When calling edit the function _VFOrmatFile was being redefined and it failed(?)